payment_gateway/models/transaction_mixin.py

```diff
@@ -39,11 +39,10 @@ def _get_transaction_to_capture_amount(self):
 
     @api.multi
     def capture_transaction(self):
-        for record in self:
-            for transaction in record.transaction_ids:
-                if transaction.state == 'to_capture':
-                    amount = record._get_transaction_to_capture_amount()
-                    transaction.capture(amount)
+        transactions = self.mapped("transaction_ids").filtered(
+            lambda r: r.state == 'to_capture')
+        for transaction in transactions:
```

Comment:
why did the amount disappear ?

Reply:
Because the capture on `gateway.transaction` doesn't have any `amount` args (so it was a mistake to have it there).